### PR TITLE
[chore] vcenterreceiver: Updates mockserver responses for better VM Resource Pool attributes

### PR DESCRIPTION
**Pull request by StefanKurek**
**Description:** <Describe what has changed.>
Minor change. VMs (not templates) typically have Resource Pool attributes associated with them, and the mockserver responses were not reflecting this for the unit tests. Updated the tests to reflect this.

**Link to tracking Issue:** <Issue number if applicable>
N/A

**Testing:** <Describe what testing was performed and which tests were added.>
Unit tests updated and still working

**Documentation:** <Describe the documentation added.>
N/A

### CODE REVIEW

File: .chloggen/vcenterreceiver-update-mockserver.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```
Comment (djaglowski) [inline .chloggen/vcenterreceiver-update-mockserver.yaml:2]: Appreciate you adding a changelog but we actually typically do not for test-only changes. I've added the `Skip Changelog` label to the PR so the check will pass if you remove it.

Comment (StefanKurek) [reply .chloggen/vcenterreceiver-update-mockserver.yaml]: @djaglowski makes sense. I'll wait for this batch of tests to finish then I'll remove this file.

**Review by djaglowski — APPROVED** [review]

File: .chloggen/vcenterreceiver-update-mockserver.yaml (hunk 1 above)
Comment (djaglowski) [reply .chloggen/vcenterreceiver-update-mockserver.yaml]: Sorry, didn't see your note. I was able to delete it directly. Should be all set.

**Comment by djaglowski** [comment]
Thanks @StefanKurek

**Pull request merged**